checkQC/default_config/config.yaml

```diff
@@ -1,21 +1,6 @@
 
 
-# Usage instruction for config
-# -----------------------------
-#  - Please note that intervals for read lengths are specified as: min < x <= max (i.e. upper inclusive, lower exclusive)
-#  - All other intervals are exclusive.
-#  - Values that are specified under each handler, e.g.
-#
-#      - name: ClusterPFHandler
-#        warning: 180 # Millons of clusters
-#        error: unknown
-#
-#    are specific to that partiular handler, but in general any value can be substituted with "unknown", in which case
-#    this will not be evaluated.
-#
-#  - Handlers specified under "default_handlers" will be run regardless of instrument type. For all other cases
-#    it is possible to specify handlers per instrument and read length interval.
-#
+# For information about config usage, see http://checkqc.readthedocs.io/
```

Comment:
Can we make this link to the exact part in the docs? I think the headlines have anchors which should make this possible.

Reply:
I checked now. The link should be: http://checkqc.readthedocs.io/en/latest/#configuration-file

Reply:
Great, will add it.